.github/workflows/sdks.yml

```diff
@@ -480,3 +480,95 @@ jobs:
           api-level: 29
           arch: x86_64
           script: ~/test-toolchain.sh
+      # the following steps fetch the cached version of each of the SDKs
+      # and assemble and upload a merged artifact
+      - name: Get cached SDK (x86_64)
+        if: ${{ matrix.os == 'ubuntu-22.04' && matrix.arch == 'x86_64' }}
+        uses: actions/cache/restore@v4
+        with:
+          path: ~/swift-${{ matrix.version }}-android-x86_64-*-sdk.tar.xz
+          key: ${{ steps.version.outputs.tag }}-x86_64-ndk-27b-sdk
+          fail-on-cache-miss: true
+      - name: Get cached SDK (aarch64)
+        if: ${{ matrix.os == 'ubuntu-22.04' && matrix.arch == 'x86_64' }}
+        uses: actions/cache/restore@v4
+        with:
+          path: ~/swift-${{ matrix.version }}-android-aarch64-*-sdk.tar.xz
+          key: ${{ steps.version.outputs.tag }}-aarch64-ndk-27b-sdk
+          fail-on-cache-miss: true
+      - name: Get cached SDK (armv7)
+        if: ${{ matrix.os == 'ubuntu-22.04' && matrix.arch == 'x86_64' }}
+        uses: actions/cache/restore@v4
+        with:
+          path: ~/swift-${{ matrix.version }}-android-armv7-*-sdk.tar.xz
+          key: ${{ steps.version.outputs.tag }}-armv7-ndk-27b-sdk
+          fail-on-cache-miss: true
+      - name: Assemble SDK
+        if: ${{ matrix.os == 'ubuntu-22.04' && matrix.arch == 'x86_64' }}
+        id: assemble-sdk
+        run: |
+          set -x
+          ls -lah ~/swift-*-sdk.tar.xz
+
+          # e.g. swift-5.10.1-RELEASE-android-sdk
+          SDK_DIR=${{ steps.version.outputs.tag }}-android-sdk
+          echo "Merging $(echo $(ls -1)) into SDK_DIR: ${SDK_DIR}"
+
+          mkdir ${SDK_DIR}
+
+          for SDK_PATH in ~/swift-*-sdk.tar.xz; do
+            MERGEDIR=$(basename $SDK_PATH .tar.xz)
+            mkdir ${MERGEDIR}
+            pushd ${MERGEDIR}
+            # e.g. swift-android-sdk-release-aarch64/swift-release-android-aarch64-24-sdk.tar.xz
+            tar xf ${SDK_PATH}
+            tree swift-*-sdk
+            pushd swift-*-sdk
+
+            ARCH=$(ls usr/lib/swift_static/android/*/swiftrt.o | cut -f 5 -d '/')
+
+            TRIPLE="${ARCH}-linux-android"
+            if [[ $ARCH == 'armv7' ]]; then
+                TRIPLE="arm-linux-androideabi"
+            fi
+
+            echo "Creating ${TRIPLE} from ${MERGEDIR}…"
+
+            # remove the destination files, which include invalid absolute paths
+            rm usr/swiftpm-*.json
+
+            mkdir -p usr/lib/${TRIPLE}
+            rm -r usr/bin
+            mv usr/lib/swift_static/android/*.a usr/lib/swift_static/android/${ARCH}/
+            mv usr/lib/*.so* usr/lib/swift/android/*.so* usr/lib/${TRIPLE}
+            mv usr/lib/pkgconfig usr/lib/${TRIPLE}
+            #mv usr/lib/swift/pm usr/lib/${TRIPLE}
+            rm -r usr/lib/swift/pm
```

Comment:
This is correct, remove the comment above. I only install this directory with `--install-llbuild` when building the SDK because it is required to build SwiftPM, which I only build to test cross-compiling with the newly-built SDK.

You'll notice that I removed this directory in previous muti-arch SDKs, like 5.10, also.

Reply:
Removed in be15a11